ts/features/itwallet/presentation/remote/machine/guards.ts

```diff
@@ -1,2 +1,11 @@
-// This file is a placeholder for future implementation
-export const createRemoteGuardsImplementation = () => ({});
+import { useIOStore } from "../../../../../store/hooks.ts";
+import { isItwEnabledSelector } from "../../../common/store/selectors/remoteConfig.ts";
+import { itwLifecycleIsValidSelector } from "../../../lifecycle/store/selectors";
+
+export const createRemoteGuardsImplementation = (
+  store: ReturnType<typeof useIOStore>
+) => ({
+  isItwWalletInactive: () =>
+    !itwLifecycleIsValidSelector(store.getState()) &&
+    isItwEnabledSelector(store.getState())
```

Comment:
I think it's easy to misunderstand what this guard means. To use the remote presentation we must check if the wallet can be used, so if it is in a valid state and the FF is enabled. What do you think about inverting it's meaning for more clarity?

```suggestion
  isWalletActive: () =>
    itwLifecycleIsValidSelector(store.getState()) &&
    isItwEnabledSelector(store.getState())
```

This way we know when a wallet is in a valid state and can be used.